### PR TITLE
LSR-915 Improvements to AssetDisplayScreenlet

### PR DESCRIPTION
**Pull request by dgarciasarai**
- Allow to load AssetDisplayScreenlet by assetEntry object.
- Allow to load an ImageEntry (not decorated AssetEntry).

### CODE REVIEW

File: ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift [hunk 1]
```diff
@@ -122,10 +122,16 @@ import UIKit
 
 	public func configureInnerScreenlet(innerScreenlet: BaseScreenlet, asset: Asset) {
 		if let screenlet = innerScreenlet as? FileDisplayScreenlet {
-			screenlet.fileEntry = FileEntry(attributes: asset.attributes)
 			screenlet.autoLoad = false
 			screenlet.offlinePolicy = self.offlinePolicy
 			screenlet.presentingViewController = self.presentingViewController
+
+			if let image = asset as? ImageEntry {
+				screenlet.classPK = image.imageEntryId
```
Comment (jmnavarro) [inline ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift]: @dgarciasarai no need to set the className?

File: ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift [hunk 2]
```diff
@@ -122,10 +122,16 @@ import UIKit
...
+			} else {
```
Comment (jmnavarro) [inline ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift]:
```
if xx {
}
else {
}
```

Comment (dgarciasarai) [inline ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift]: done 👍

File: ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift (hunk 1 above)
Comment (dgarciasarai) [inline ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift]: The className is set by default in FileDisplayScreenlet so you don't need to set it here.

**Comment by jmnavarro** [comment]
:+1 for iOS. Please @ahdezma merge when the Android code is approved

**Comment by ahdezma** [comment]
👍 for Android too. Merging now.

**Pull request merged**